src/platform/STM32/sdio_h7xx.c

```diff
@@ -160,7 +160,11 @@ void sdioPinConfigure(void)
 
 #undef SDIOFINDPIN
 
+#if defined(SDIO_USE_PULLUP) && SDIO_USE_PULLUP
```

Comment:
```suggestion
#if defined(SDIO_USE_PULLUP) && defined(SDIO_USE_PULLUP)
```

Reply:
lol :facepalm: clearly in a hurry. Thanks!

Reply:
Just need the one... unless there's a typo

Reply:
@dakejahl please also rename to `USE_SDIO_PULLUP` for convention.